### ISSUE TITLE
default smbclient const not working

### ISSUE THREAD

**Issue by jza34** [owncloud/apps#2054]
By default, in "owncloud/apps/user_external/lib/smb.php", the smbclient command is defined with:
```const SMBCLIENT = 'smbclient' ```
Well it is enough but it is not working. Smbclient needs a command like "-L" to be ok.

Everytime I upgrade owncloud I have to do the change: 
```
- const SMBCLIENT = 'smbclient'
+ const SMBCLIENT = 'smbclient -L'
```
This little change makes Samba4 authentication working.

I use Debian servers.
Thanks all!

**Comment by MorrisJobke** [comment]
cc @icewind1991

**Comment by fbartels** [comment]
This fixes #1974 as well, as no NT_STATUS_BAD_NETWORK_NAME is thrown with "smbclient -L"
